lang/de_DE.po

```diff
@@ -14766,7 +14780,7 @@ msgid ""
 msgstr ""
 
 msgid "IPv4 Remote Network/s"
-msgstr ""
+msgstr "IPv4 Remote Netzwerk(e)"
```

Comment:
Ja, denke ich auch, deswegen nur "fern". Weiß nicht, ob Lokalnetzwerke / Fernnetzwerke so passt. Na, die Rechtschreibung in OSX mag die Begriffe. Von mir aus gern.